### ISSUE TITLE
Allow users to specify their own OpenFoodFacts account

### ISSUE THREAD

**Issue by grrrrr** [davidhealey/waistline#50]
At the moment Waistline uses the generic OpenFoodFacts account to add new content to OFF.  It would be a goodto have an option allowing users to enter their own existing OFF account and if not specified drop back to the default user

**Comment by davidhealey** [comment]
I have no current plans to add this feature but I will add it to the low priority list. The OFF integration is secondary to the app's intended purpose, in fact I only added the upload from within the app because people were asking me to, I was happy to use the OFF official app. Although it is nice to have the built in integration and I use it a lot. Having the generic user account is helpful because it allows us to see which data has been added through the app and make improvements, for example looking at the OFF database and the contributions by Waistline users I can see there is an image upload issue, so I'll try and fix that. If someone really wants to use their own log in details then they can recompile the app and swap out the credentials for now.
